apim-operator/deploy/analytics-configs/analytics_pvc_template.yaml

```diff
@@ -18,7 +18,7 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: analytics-pv-claim
-  namespace: wso2-system
+  namespace: default
```

Comment:
No need to hard code these values..

Reply:
removed this files, since we use emptydir volume to mount